### PR TITLE
Feat: Like and dislike feature

### PR DESCRIPTION
**Pull request by pritam-kr**
- Filter by recent, sort by data, trending post
- created avatar component for users
- [Feat: Delete post of a user](https://github.com/pritam-kr/like/pull/9/commits/4f82ea0a7c1f5f383b4d4fd8e47c74738d88546d)
- [Feat: Like and dislike feature](https://github.com/pritam-kr/like/pull/9/commits/69e80cbf02f153eccb0eb02062b88433155f43e3)

### CODE REVIEW

**Review by himadri2110 — APPROVED** [review]
The like/dislike feature does not reflect immediately on the home page. It works well on the profile page.
Suggestion: You can show the active state of the applied filter (from trending, sort by, and recent posts)

The delete post feature works nicely.
Good work!

File: src/Hooks/usePost.js [hunk 1]
```diff
@@ -25,7 +23,8 @@ export const usePost = () => {
   //Fetching post Data for a specific user from database
   useEffect(() => {
     dispatch(getUserPosts({username: username }))
```
Comment (gopalbbharadva) [inline src/Hooks/usePost.js:25]: Suggestion: if object's key and value is same then we can write it once only.

File: src/Hooks/usePost.js [hunk 2]
```diff
@@ -57,33 +56,33 @@ export const usePost = () => {
...
+  //       // Doing filter data on the basis of user
+  //       const filteredData = posts.filter(
+  //         (eachPost) => eachPost.username === `${username}`
+  //       );
+  //       dispatch(getLikePost(filteredData));
+  //     }
+  //   } catch (error) {
+  //     console.log(error);
+  //   }
+  // };
```
Comment (gopalbbharadva) [inline src/Hooks/usePost.js]: Remove comments

**Review by gopalbbharadva — APPROVED** [review]
Everything is good. 
Bug: Like of a post is not updating on feed page, but it's updating in user profile page. just check once.

**Pull request merged**